src/components/skeletons/ProductCardSkeleton/product-card-skeleton.module.scss

```
@@ -0,0 +1,107 @@
+@import "src/styles/scaffold";
+
+.fs-product-card-skeleton {
+  // --------------------------------------------------------
+  // Design Tokens for Product Card Skeleton
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-product-card-skeleton-min-width-mobile               : 14vw;
+  --fs-product-card-skeleton-min-width-desktop              : 12rem;
+  --fs-product-card-skeleton-height                         : 100%;
+  --fs-product-card-skeleton-padding                        : var(--fs-spacing-1) var(--fs-spacing-1) var(--fs-spacing-2);
+  --fs-product-card-skeleton-border                         : var(--fs-border-width) solid transparent;
+  --fs-product-card-skeleton-border-radius                  : var(--fs-border-radius);
+
+  // Image
+  --fs-product-card-skeleton-image-height-mobile            : 45vw;
+  --fs-product-card-skeleton-image-height-desktop           : #{rem(195px)};
```

I think it doesn't work, in other parts we are converting manually
```suggestion
  --fs-product-card-skeleton-image-height-desktop           : 12.188rem;
```